backend/README.md

```diff
@@ -17,7 +17,7 @@ Please visit this page for more details:
 [https://docs.docker.com/compose/compose-application-model/](https://docs.docker.com/compose/compose-application-model/)
 
 ```
-$ docker compose up
+$ docker compose up --build
```

Comment:
The `--build` option forces Docker Compose to rebuild images, and it's required when we change Docker-related configuration, but for typical use cases such as code changes, rebuilding every time downloads images from DockerHub and uses much memory space to build them. Therefore, we should avoid using it if not required.

Because not everyone is familiar with Docker, updating the README this way might confuse them into thinking that the flag is always required. I would suggest adding another section to explain the `--build` option, or because the option will be used when changing Docker configuration, I think we don't have to add it here.

Reply:
okay I will fix it.